src/rss-diff.js

```diff
@@ -1,5 +1,24 @@
-function rssNDiff(itemBefore, itemAfter, field) {
-    let delta
-}
+/**
+* Returns the added packages between two rss feed captures
+* @param itemBefore {Array} The first rss capture
+* @param itemAfter {Array} The second rss capture
+* @param field {String} The field of each rss item to diff off of
+* @return {Array} The packages added between the two captures
+*/
+module.exports = function rssDiff(itemBefore, itemAfter, field) {
+    let beforeMap = {}
+    let out = []
+
+    // Create a map of field to index for first array
+    for (let i = 0; i < itemBefore.length; i++) {
+        beforeMap[itemBefore[i][field]] = true
+    }
 
-exports.rssDiff = rssNDiff
+    // If element in second array does not exist in previous map, store its index
+    for (let i = 0; i < itemAfter.length; i++) {
+        if (!(itemAfter[i][field] in beforeMap)) {
+           out.push(itemAfter[i])
```

Comment:
Ah, I replied to the last comment, but the code got changed.

As far as I know Pypi doesn't allow package removal (I remember reading discussion about it when the npm left-pad fiasco happened). I'll create a test package and make sure though.

In any case the rss feed is always just the last 40 packages updated in Pypi, the package not appearing in the second rss capture isn't necessarily removed from Pypi, it might have just been knocked off the latest package list to make room for an update to another package.